src/content/1.7/code/agda/snippet32.agda

```diff
@@ -0,0 +1 @@
+fmap : (A → B) → f A → f B
```

Comment:
Any particular reason for lower case f instead of upper case?

Reply:
...because `snippet32.hs` uses lower case `f`... but I agree an upper case `F` seems more natural here.  I'll change it.

Reply:
Actually, for consistency with the paragraph in the text below this code snippet, where the author specifically refers to *lower-case* `f`, and where he uses lower-case `a` and `b`, I think I should stick with lower-case `f` and change the upper-case `A` and `B` to lower-case `a` and `b`.

Thereafter, however, I think we should generally use capital letters for types in Agda and a capital `F` for the type constructor functor.

Reply:
However, I will stick to lower case whenever such a type variable is mentioned in the prose surrounding the code.